sidebars/core-concepts.js

```diff
@@ -30,7 +30,7 @@ export const coreConcepts = [
   "developer/checkout/api-guide",
   "developer/checkout/address",
   "developer/checkout/problems",
-  "developer/checkout/finalizing",
+  "developer/checkout/cookbook",
```

Comment:
Probably also not a scope of these changes, but shouldn't we have api-guide at the end of the section? 
![image](https://github.com/user-attachments/assets/35239c12-bf80-4b26-834d-02cec917db2e)
We have api-guide for products and attributes at the end of the section, so I assume that the same thing should be for checkout part